blazor/dropdown-list/localization.md

```diff
@@ -7,6 +7,113 @@ control: DropDown List
 documentation: ug
 ---
 
-# Localization in Blazor DropDown List Component
+# Localization in Dropdown List
```

Comment:
Remove all content and codes just redirect to the common topic (Refer RTE control) -  https://blazor.syncfusion.com/documentation/rich-text-editor/globalization#localization